Makefile

```diff
@@ -63,7 +63,7 @@ it-sync: build
 
 it-polkadotjs: build
 	@echo "  >  \033[32mRunning Integration Tests polkadot.js/api mode...\033[0m "
-	HOSTNAME=0.0.0.0 MODE=polkadot go test ./tests/polkadotjs_test/... -timeout=5m -v
+	HOSTNAME=0.0.0.0 MODE=polkadot go test ./tests/polkadotjs_test/... -timeout=5m -v -test.v
```

Comment:
What does `-test.v` do? :thinking:

Reply:
Is similar to `-v` but I'm not sure ()https://pkg.go.dev/testing#T.Log)